apps-devstg/us-east-1/dynamodb-access-rules-demoapps --/demo_client/run_demo.sh

```
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# --- Configuration ---
+PYTHON_SCRIPT="main.py" # ⚠️ Replace with the actual name of your Python script
+# ---------------------
+
+echo "🚀 Fetching Terraform outputs and setting environment variables..."
+
+cd /home/jdelacamara/Dev/work/BinBash/code/le-tf-infra-aws/apps-devstg/us-east-1/research-dynamodb-access-rules/
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)